src/cleed++/Util/teestream.hh

```
@@ -0,0 +1,16 @@
+#ifndef TEESTREAM_HH
+#define TEESTREAM_HH
+
+#include "teebuf.hh"
+
+class teestream : public std::ostream
+{
+public:
+    // Construct an ostream which tees output to the supplied
+    // ostreams.
+    teestream(std::ostream & o1, std::ostream & o2);
```

Clang (14.0.6)

[warning] parameter name 'o1' is too short, expected at least 3 characters

(readability-identifier-length)